lib/capistrano/tasks/invoke.rake

```diff
@@ -1,9 +1,12 @@
 namespace :invoke do
 
-  desc "Execute a rake task on a remote server"
+  # Defalut to :app roles
+  rake_roles = fetch(:rake_roles, :app)
+
+  desc "Execute a rake task on a remote server (cap invoke:rake TASK=db:migrate)"
   task :rake do
     if ENV['TASK']
-      on roles(:app) do
+      on roles(fetch(:rake_roles)) do
```

Comment:
Shouldn't this be `rake_roles` instead of `fetch(:rake_roles)`?

Reply:
Yes, good eye😉